Resources/views/Pagination/filtration.html.twig

```diff
@@ -1,10 +1,14 @@
 <form method="get" action="{{ action }}" enctype="application/x-www-form-urlencoded">
 
-    <select name="{{ filterFieldName }}">
-        {% for field, label in fields %}
-            <option value="{{ field }}"{% if selectedField == field %} selected="selected"{% endif %}>{{ label }}</option>
-        {% endfor %}
-    </select>
+    {% if fields|length > 1 %}
+        <select name="{{ filterFieldName }}">
+            {% for field, label in fields %}
+                <option value="{{ field }}"{% if selectedField == field %} selected="selected"{% endif %}>{{ label }}</option>
+            {% endfor %}
+        </select>
+    {% else %}
+        <input type="hidden" name="filterField" value="{{ fields|keys|first }}" />
```

Comment:
I guess that here it should be `{{ filterFieldName }}` instead of `filterField`, no?

Reply:
You're right, same in bootstrap4 template see my separate pull request #561